apidoc/Titanium/UI/View.yml

```diff
@@ -1323,6 +1323,13 @@ properties:
     type: Number
     default: 0
 
+  - name: borderRadiusCorners
+    summary: An array of corners to use specified <Titanium.UI.View.borderRadius> on.
+    type: Array<Number>
+    constants: Titanium.UI.BORDER_CORNER_*
+    platforms: [android, iphone, ipad]
+    since: "9.1.0"
+
```

Comment:
This feature is only supported on Android 5.0 and above, right?
In which case we need to add the following...
`osver: {android: {min: "5.0"}}`

Reply:
We should also document that if this property is "undefined", then `borderRadius` is applied to all corners. If an array is assigned to this property, then it must specify each corner to apply to radius to. If assigned an empty array, then `borderRadius` is not applied to any other corners.